source/testcommon/testcommon/dataframes/write_to_delta.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
Everything strictly related to scenario tests should IMHO be in etl.
Perhaps we should rename etl to `scenario_testing` or something else more helpful (in another PR).